bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo_de.properties

```diff
@@ -42,28 +42,31 @@ thing-type.config.netatmo.bridge.readThermostat.description = Aktiviert den Zugr
 
 # thing types
 thing-type.netatmo.NAMain.label = Haupt-Indoor-Modul
-thing-type.netatmo.NAMain.description = Das Haupt-Indoor-Modul liefert Daten wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit, CO2-Gehalt und Lautst�rke.
+thing-type.netatmo.NAMain.description = Das Haupt-Indoor-Modul liefert Daten wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit, CO2-Gehalt und Lautst�rke.
 
 thing-type.netatmo.NAModule1.label = Outdoor-Modul
 thing-type.netatmo.NAModule1.description = Das Outdoor-Modul liefert Daten wie z.B. Temperatur und Luftdruck.
 
 thing-type.netatmo.NAModule2.label = Windmesser
-thing-type.netatmo.NAModule2.description = Der Windmesser liefert Daten wie z.B. Windrichtung und Windst�rke.
+thing-type.netatmo.NAModule2.description = Der Windmesser liefert Daten wie z.B. Windrichtung und Windst�rke.
 
 thing-type.netatmo.NAModule3.label = Regenmesser
-thing-type.netatmo.NAModule3.description = Der Regenmesser liefert Daten wie z.B. Echtzeit-Messungen zur Intensit�t des Regens.
+thing-type.netatmo.NAModule3.description = Der Regenmesser liefert Daten wie z.B. Echtzeit-Messungen zur Intensit�t des Regens.
 
 thing-type.netatmo.NAModule4.label = Zusatz-Indoor-Modul
 thing-type.netatmo.NAModule4.description = Das Zusatz-Indoor-Modul liefert Daten wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit und CO2-Gehalt.
 
 thing-type.netatmo.NAPlug.label = Thermostat Relais
 thing-type.netatmo.NAPlug.description = Das Thermostat Relais liefert Daten des Heizkessels.
 
-thing-type.netatmo.NATherm1.label = Heizk�rperthermostat
-thing-type.netatmo.NATherm1.description = Das Heizk�rperthermostat dient zur Steuerung von Heizk�rpern und liefert Daten wie z.B. Temperatur.
+thing-type.netatmo.NRV.label = Heizkörper Ventil
+thing-type.netatmo.NRV.description = Das Ventil steuert den Heizkörper.
+
+thing-type.netatmo.NATherm1.label = Thermostat
+thing-type.netatmo.NATherm1.description = Das Thermostat dient zur Steuerung von Räumen und liefert Daten wie z.B. Temperatur.
 
 thing-type.netatmo.NHC.label = Healthy Home Coach
-thing-type.netatmo.NHC.description = Der Healthy Home Coach liefert Daten wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit, CO2-Gehalt und Lautst�rke.
+thing-type.netatmo.NHC.description = Der Healthy Home Coach liefert Daten wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit, CO2-Gehalt und Lautst�rke.
```

Comment:
I think there is a code page error

Reply:
Hmm, it is set here as UTF-8 (VSCode on MacOS). What encoding should these have ?

Reply:
You're right, I see same encoding on my fork of this file, so maybe the original encoding was wrong. If I change it to ISO 8859-1 it seems correct.

Reply:
Changed to UTF-8 and used umlauts consistenly. Looks good on Github